pypredict/api/endpoints/about.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from fastapi import Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+
+templates = Jinja2Templates(directory="templates")
+router = APIRouter()
+
+@router.get("/about")
+async def about(request: Request):
```

Comment:
Interesting, an async framework will probably work well for interacting with the Python SDK.